src/software/thunderscope/gl/layers/gl_world_layer.py

```diff
@@ -460,10 +475,18 @@ def __update_robot_graphics(
                 color=Colors.PRIMARY_TEXT_COLOR,
             ),
         )
+        robot_name_graphics.resize(
+            len(robots),
+            lambda: GLTextItem(
+                font=QtGui.QFont("Roboto", 10, weight=700),
```

Comment:
nit: `QtGui.QFont("Roboto", 10, weight=700)` could be declared as a constant `GLWorldLayer.TEXT_GRAPHICS_QFONT)` that could be shared between `robot_id_graphics` and `robot_name_graphics`

Reply:
Should I define this in the constants.py file, or as a constant for the glWorldLayer Class?

Reply:
I'd prefer it as a constant of the glWorldLayer class. In the future, if we end up duplicating the fonts in other layers, I would move it to constants

Reply:
So it should be defined at the top of that file?

Reply:
yup